src/Illuminate/Mail/PendingMail.php

```diff
@@ -114,12 +114,11 @@ public function bcc($users)
      * Send a new mailable message instance.
      *
      * @param  \Illuminate\Contracts\Mail\Mailable  $mailable
-     *
-     * @return mixed
+     * @return void
```

Comment:
Here I removed the extra blank line between parameters and return, and also changed the return to match the interface.

Mailer implementation `send` method does not return anything.

Reply:
Oh okay, did Doctum say something about that ?

Reply:
No, but when I was reviewing the others I found this. Neither doctum or StyleCI complained about this one.

I will update my opening comment to clarify I made other fixes.

Reply:
phpstan would have got it I think ;p